### ISSUE TITLE
Draw line given two points

### ISSUE THREAD

**Issue by sank96** [openseadragon/openseadragon#2410]
Hi, I would like to draw on an image a line given the two vertices. I have already plotted the points, but I'm not able to add a line between them.
I tried to search for something similar in the repo without any success. 
Can you help me to understand if it is possible or not?  If yes, how?

**Comment by pearcetm** [comment]
There are a variety of [plugins available](https://openseadragon.github.io/#plugins) that can do such things. If you haven't had a look at these yet, that would be a good next step.

**Comment by RaunakGangi** [comment]
hi @sank96 
i'm stuck with the same problem, can you help me with it if you've solved the problem. Thanks in advance

**Comment by sank96** [comment]
@RaunakGangi no,I don't have checked the list of plugins given by pearcetm yet. If I will reach some improvements I will update you.

**Comment by RaunakGangi** [comment]
@sank96 okay but are you able to plot two points on the image? can u give me some insight on how you're doing that if you git it

**Comment by Aiosa** [comment]
You have two options: vector and raster. Rasterizing a line on the image can be done with `tile-loaded` event and `Canvas` API (`lineTo`). But you would have to do your math per tile to check which tiles cross the line and how.
Second option is to go vector. In the plugin list, you can find `fabricjs` or `paperjs` that allow you to draw vector data atop the OSD canvas. FabricJS plugin is a bit old and has some issues, but should do the trick. Note it cannot deal with the OSD rotation features, also on my todo list ;) Will update the plugin once I get into it.
